epigraphhub/models/forecast_swiss.py

```diff
@@ -5,6 +5,9 @@
 the data in train and test datasets), train with all the data available, and make
 forecasts. Also, there are functions to apply these methods in just one canton or all
 the cantons of switzerland. 
+
+All the functions assigned with `article` at the end of the name use data saved in the data folder. 
+It was done to allow the reproduction of the results of the article.  
```

Comment:
I think the best solution here is to move these function to the article notebooks, so that it doesn't "polute" the API of the library.